cmake/FreecivInstall.cmake

```diff
@@ -22,12 +22,10 @@ install(
   ${CMAKE_SOURCE_DIR}/dist/licenses/FTL.txt
   ${CMAKE_SOURCE_DIR}/dist/licenses/GPL2.txt
   ${CMAKE_SOURCE_DIR}/dist/licenses/GPL3.txt
-  ${CMAKE_SOURCE_DIR}/dist/licenses/IMAGEMAGICK.txt
```

Comment:
I think we should delete this file too.

Reply:
My bad, I missed that there were two licenses for ImageMagick